packages/manager/apps/dedicated/client/app/billing/main/history/legacy/billing-main-history.html

```diff
@@ -0,0 +1,306 @@
+<div class="billing-main-history" data-ui-view>
+    <h2 data-translate="billing_main_history_legacy_title"></h2>
+
+    <div class="text-center" data-ng-if="$ctrl.loading.init">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div data-ng-if="!$ctrl.loading.init">
+        <div data-ng-if="$ctrl.postalMailOptions.enabled">
+            <oui-checkbox
+                class="d-inline-block ml-3"
+                data-text="{{ :: 'billing_main_history_legacy_invoices_postal_mail' | translate }}"
+                data-on-change="$ctrl.onPostalMailOptionsChange()"
+                data-model="$ctrl.postalMailOptions.activated"
+            >
+            </oui-checkbox>
+        </div>
+
+        <div
+            data-ng-if="$ctrl.debtAccount.active && ($ctrl.debtAccount.dueAmount.value > 0 || $ctrl.debtAccount.todoAmount.value > 0)"
+        >
+            <p
+                class="text-warning"
+                data-ng-if="!$ctrl.hasDefaultPaymentMethod && $ctrl.debtAccount.dueAmount.value > 0"
+                data-translate="billing_main_history_legacy_debt_to_pay"
+                data-translate-values="{ t0: $ctrl.debtAccount.dueAmount.text }"
+            ></p>
+
+            <p
+                class="text-info"
+                data-ng-if="$ctrl.hasDefaultPaymentMethod && $ctrl.debtAccount.dueAmount.value > 0"
+                data-translate="billing_main_history_legacy_debt_automatically_collected"
+                data-translate-values="{ t0: $ctrl.debtAccount.dueAmount.text }"
+            ></p>
+
+            <p
+                class="text-warning"
+                data-ng-if="!$ctrl.hasDefaultPaymentMethod && $ctrl.debtAccount.todoAmount.value > 0"
+                data-translate="billing_main_history_legacy_debt_to_pay"
+                data-translate-values="{ t0: $ctrl.debtAccount.todoAmount.text }"
+            ></p>
+
+            <p
+                class="text-info"
+                data-ng-if="$ctrl.hasDefaultPaymentMethod && $ctrl.debtAccount.todoAmount.value > 0"
+                data-translate="billing_main_history_legacy_debt_automatically_collected"
+                data-translate-values="{ t0: $ctrl.debtAccount.todoAmount.text }"
+            ></p>
+        </div>
+
+        <!-- @TODO: remove this datagrid when "dynamic" columns will allow transclusion -->
```

Comment:
You can remove that, it will never be done

Reply:
I tried to update the legacy the bare minimum but sure

Reply:
I know, it just makes me sad to see `TODO` that will never be resolved